CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(geometric_shapes)
 
+add_compile_options(-std=c++11)
```

Comment:
I seem to remember that `add_compile_options()` is a `>=2.8.12` thing. Should the minimum cmake version be increased to reflect that?